app/views/topics/show.js.erb

```diff
@@ -0,0 +1,2 @@
+document.title = "<%= j page_title %>";
+$("#tab-content").html("<%= j html(@topic.description) %>");
```

Comment:
Could be `$("#tab-content").html("<%= partial 'topics/_description, topic: @topic %>");`, where that partial looks like
```
<h3>Learning Objectives</h3>
<ul>
  <% topic.objectives.each do |objective| %>
    <li><%= objective %></li>
  <% end %>
</ul>
<# etc #>
```
I find the `html` method confusing in general.  It seems like it's building a little template, but why not use Rails' built-in templating functionality?

Reply:
Vivian helped me to understand that it's because Admins should have freedom and flexibility in the markup they create, and we can't expect a list of learning objectives for example.